### ISSUE TITLE
[2.1.0-dev.14] onvif.mediaSupportLevel

### ISSUE THREAD

**Issue by LODYZ** [faithoflifedev/easy_onvif#37]
Hi, I noticed that there is a mediaSupportLevel property in the Onvif class. In my case this property is always set to none.

I added this check in initialize():

```
...
if(media1!=null && media2!=null){
  mediaSupportLevel=MediaSupportLevel.both;
}else if(media2!=null && media1==null){
  mediaSupportLevel=MediaSupportLevel.two;
}else if(media2==null && media1!=null){
  mediaSupportLevel=MediaSupportLevel.one;
}
...
```

Do you think it can be useful or are there better ways to know the supported media profile?

For example, at the moment i'm using that property to check the media supported level and after to call the correct getSnapshotUri method. In that case I'don't have a MixedProfile instance (profileToken is a String saved locally).

```
    if(onvif.mediaSupportLevel==MediaSupportLevel.both || onvif.mediaSupportLevel==MediaSupportLevel.two){
      snapshotUri= await onvif.media.getSnapshotUri(profileToken);
    }else{
      snapshotUri= await onvif.media.media1.getSnapshotUri(profileToken).then((value) => value.uri);
    }
```

Thanks in advance

**Comment by faithoflifedev** [comment]
I've just published 2.1.0-dev.15

In this version the following code will return a `String` regardless of the `MediaSupportLevel` of the device.

```dart
// get snapshot Uri
var snapshotUri = await onvif.media.getSnapshotUri(profs.first.token);

print(snapshotUri);

// get stream uri
var streamUri = await onvif.media.getStreamUri(profs.first.token);

print(streamUri);
```

In the case that you need to check the `mediaSupportLevel` as in your code example, you can now do this:

```dart
    if(onvif.media.mediaSupportLevel==MediaSupportLevel.both || onvif.media.mediaSupportLevel==MediaSupportLevel.two){
      snapshotUri= await onvif.media.getSnapshotUri(profileToken);
    }else{
      snapshotUri= await onvif.media.media1.getSnapshotUri(profileToken).then((value) => value.uri);
    }
```

Though personally I would write the code this way:

```dart
if (onvif.media.mediaSupportLevel==MediaSupportLevel.one) {
  snapshotUri = (await onvif.media.media1.getSnapshotUri(profileToken)).uri
} else {
  snapshotUri= await onvif.media.media2.getSnapshotUri(profileToken);
}
```

**Comment by LODYZ** [comment]
Hi @faithoflifedev, I've tested the latest release and it works. Thank you
